rs/sns/integration_tests/src/manage_dapp_canister_settings.rs

```diff
@@ -319,5 +319,9 @@ fn test_manage_dapp_canister_settings_failure() {
         )
         .unwrap()
         .unwrap();
-    assert_eq!(new_ledger_canister_status, original_ledger_canister_status);
+    // ignore the canister version in the comparison
```

Comment:
Why?

Reply:
A few lines above there's a comment

> Verify that the ledger canister settings have not been changed.

so we should extract the settings out of the canister status for the sake of comparison instead of comparing the entire canister status (now also containing a fine-grained canister version which is expected to change).